include/glow/Importer/CommonOperatorLoader.h

```diff
@@ -1093,7 +1093,18 @@ class CommonOperatorLoader : public ProtobufLoader {
       batchDims = axis;
     }
 
-    Node *GN = G_->createGather(loadOperatorName(op), data, indices, batchDims);
+    Node *GN = nullptr;
```

Comment:
nit: Don't need this up here anymore since it isn't used until assigned via `createGather()` below.